source/navitiacommon/navitiacommon/models.py

```diff
@@ -432,6 +432,37 @@ def delete_dataset(self):
 
         db.session.commit()
 
+    def delete_old_jobs(self, time_limit):
+        """
+        Delete jobs created before the date parameter 'time_limit'
+        :param time_limit: date from which jobs will be deleted
+        :return: list of datasets to delete
+        """
+        # Keep the last dataset of each type to be able to reload data
+        dataset_to_keep = self.last_datasets()
```

Comment:
We may parametrize it a bit to keep more than one dataset?
This can be done in another PR if it's a good idea.